ap_api/resources/quotes/index.mdx

```diff
@@ -5,7 +5,11 @@ order: 0
 
 import {EndpointsTable} from "@site/src/components/EndpointsTable";
 
-The quote endpoint is used to provide potential trading pairs and exchange rates for a given asset pair. For example, if a user wants to trade their USDC for XLM, the quote endpoint would provide them with the current exchange rate for that trade.
+The `/quotes` endpoints are used to fetch exchange rates previously provided to users via the [SEP-38 RFQ API][sep-38] implemented by the Anchor Platform. Quotes are commitments to trade an off-chain asset such as fiat euro for an on-chain asset such as USDC, or vice-versa, at a specified rate. Quotes are only valid for a specified period of time.
```

Comment:
Wording suggestions for clarity, take it or leave it.

"The /quotes endpoints fetch exchange rates previously provided to users via the [SEP-38 RFQ API](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0038.md) implemented by the Anchor Platform. Quotes are commitments to trade an off-chain asset (such as the fiat euro) for an on-chain asset (such as USDC) or vice-versa at a specified rate. Quotes are only valid for a specified period of time."